README.md

```diff
@@ -178,6 +178,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [ ] [angular](https://github.com/steelsojka/tree-sitter-angular) (experimental)
 - [x] [apex](https://github.com/aheber/tree-sitter-sfapex) (maintained by @aheber)
 - [x] [arduino](https://github.com/ObserverOfTime/tree-sitter-arduino) (maintained by @ObserverOfTime)
+- [x] [asm](https://github.com/rush-rs/tree-sitter-asm) (maintained by @RubixDev, @WillLillis)
```

Comment:
Have these volunteered to maintain the **queries** in nvim-treesitter?

Reply:
(I'm not sure we want to accept further parsers without active query maintainers...)

Reply:
They are the authors of [tree-sitter-asm](https://github.com/rush-rs/tree-sitter-asm). @RubixDev, @WillLillis

Reply:
authors of tree-sitter-asm doesn't mean they will maintain asm queries for us

Reply:
> They are the authors of [tree-sitter-asm](https://github.com/rush-rs/tree-sitter-asm). @RubixDev, @WillLillis

Yes, that's my point. That is **not** what this field means.

Reply:
OK, I know both of them are tree-sitter and nvim users. If they don't agree. I'll take duty of it.

Reply:
You gotta ask _first_ before "volunteering" them... We won't merge this without an explicit agreement from the listed maintainer (if not equal to the person making the PR).

Reply:
Because I thought the maintainer means the maintainers of tree-sitter-XXX :cry:

Reply:
What would be the point of adding that information in _our_ database? It's clear from the repo link.

Reply:
> Have these volunteered to maintain the **queries** in nvim-treesitter?

@clason I am the creator and maintainer of this tree-sitter-asm parser, and I'd be willing to maintain the queries.

I was also planning to add the parser here myself, but it seems I waited too long and @Freed-Wu beat me to it 😅.

Reply:
Thank you, that is appreciated!

Reply:
Just commenting here to agree with @RubixDev. I've contributed some small fixes to the parser (and would like to continue doing so) but I'm not a maintainer. Very excited to see this getting added however! :)